.github/workflows/android.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         run: ./gradlew markdownlint
 
       - name: Unit tests
-        run: ./gradlew jacocoTestReport
+        run: ./gradlew test
```

Comment:
`test` is finalised by `jacocoTestReport` but the task doesn't by default depend on `test`